emnify/manifest.json

```diff
@@ -0,0 +1,77 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "000307b4-8304-424a-8378-daf9a41b4d93",
+  "app_id": "emnify",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitors and dashboard for EMnify data usage metrics",
+    "title": "EMnify",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "All-in-one Dashboard enables you to observe organisation consumption and analyze usage patterns of your devices inventory.",
+        "image_url": "images/data_usage_dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Monitor when Data Exchange has stopped unexpectedly for a device.",
+        "image_url": "images/datastop_monitor.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Forecast and notify when data consumption would have gone above expected.",
+        "image_url": "images/above_forecast_monitor.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Be notified when Data Receiving for a device is higher than usual.",
+        "image_url": "images/high_data_monitor.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Device has Data Transmission higher than usual be notified with this monitor.",
+        "image_url": "images/high_transmission_monitor.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Monitoring",
+      "Category::Metrics",
+      "Category::Cost Management"
+    ]
+  },
+  "author": {
+    "support_email": "mail@emnify.com",
+    "homepage": "https://emnify.com",
+    "sales_email": "sales@emnify.com",
+    "name": "EMnify"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "EMnify",
+      "configuration": {},
+      "service_checks": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "emnify.",
```

Comment:
@buraizu I'm a bit concerned with what we put here.

Our current solution does not prefix metrics at the moment, 
<img width="1006" alt="image" src="https://user-images.githubusercontent.com/19877114/190999609-78cc9de2-e15c-4bc1-8b6b-2e97c8986a81.png">

<img width="1620" alt="image" src="https://user-images.githubusercontent.com/19877114/190999422-7e43b1e0-889d-4517-8411-95a3e14e5755.png">

So in order to make it work should we rather not define prefix here or add it when we cast metrics instead. Or non changing it will also be fine?

Reply:
Thanks for pointing this out. @DataDog/agent-integrations may be the better party to answer this question, but I think you specify the prefix directly in the `metadata.csv` file -- at least, that's what I'm seeing in other `integrations-extras` integrations, such as [neo4j](https://github.com/DataDog/integrations-extras/blob/master/neo4j/metadata.csv).
With that being said, it looks like some of the mandatory columns may be missing data in the `metadata.csv` file, such as the mandatory `orientation` and `integration` columns mentioned in our developer docs:
https://docs.datadoghq.com/developers/integrations/check_references/#metrics-metadata-file
I hope this is helpful, but let me know if you have any questions or if I've misunderstood anything!

Reply:
This file is fine as it is, all integrations are expected to use their own prefix, and the metrics should be sent with that prefix. This avoids name clashes and makes it easier for users to find the metrics they need in Datadog.